content/attacks/posts/2021-06-23-Haven-Protocol.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2021-06-23
+categories: Protocol Exploit
+title: "Haven Protocol Hack"
+---
+
+## Summary
+
+Haven Protocol experienced a series of exploits that resulted in financial losses and raised concerns about the security of the platform. The exploits targeted various aspects of the protocol, including miner rewards, transaction validation, and conversion fees.
+
+## Attackers
+
+The identity of the attackers remains unknown.
+
+## [Losses](https://cdn.havenprotocol.org/app/uploads/2021/07/Technical-Overview-of-June-2021-Exploits.pdf)
+
+- Approximately 125.66 xBTC with an estimated value of around $4,335,000 was obtained through the exploits.
+- An amount of 202,920 xUSD, equivalent to approximately $202,920.
+- A total of 440,000 XHV with an approximate value of $1,540,000 was frozen in KuCoin.
+- TradeOgre froze 100,000 XHV, valued at approximately $350,000.
+
+## Timeline
+
+-**June 22, 2021  18:19:41:**

```

Comment:
If including the actual time is necessary, also include the relevant time zone for clarity.